README.md

```diff
@@ -1,2 +1,57 @@
-# terraform-alicloud-map-reduce
-Terraform module which creates E-MapReduce clusters on Alibaba Cloud
+# terraform-alicloud-emr-hadoop
```

Comment:
Refer to README.md in [RDS module](https://registry.terraform.io/modules/terraform-alicloud-modules/rds/alicloud/2.0.1)